starfish/image/_stack.py

```diff
@@ -45,6 +53,107 @@ def numpy_array(self, data):
         self._data_needs_writeback = True
         data.setflags(write=False)
 
+    def get_slice(
```

Comment:
rename to get_tile? we don't use the word slice in our spec doc, but i think it means tile?

Reply:
oh i see, a slice can return a tensor in addition to a matrix. ignore prevous comment